code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -715,7 +706,6 @@ butcher
 	var/list/dummy_params = list("icon-x" = 16, "icon-y" = 16)
 
 /datum/aiTask/timed/targeted/flockdrone_shoot/proc/precondition()
-	//if we know there are enemies, and gun is charged and ready
```

Comment:
useful comment

Reply:
Maybe would be better as precondition comment?